### PR TITLE
Companion ACCESS support

### PR DESCRIPTION
**Pull request by raphaelcoeffic**
TODOs:
General settings:
- [x] registrationId

Model settings:
- [x] registrationId
- [x] receiver list (`read-only` + `delete`)
- [x] telemetry sensor receiver
- [x] show receiver in various "sources" (vario, RSSI, etc)

### CODE REVIEW

**Comment by bsongis** [comment]
Yes, I think the receivers list should be readonly. Perhaps later add the possibility to remove a receiver?

**Comment by raphaelcoeffic** [comment]
> Yes, I think the receivers list should be readonly. Perhaps later add the possibility to remove a receiver?

Later? I can add just now, why not. Just not sure what it means on the TX. Is there anything to be done physically with the receiver after deleting it from the model?

**Comment by kilrah** [comment]
On the radio we have 2 options, Delete and Reset - Delete just clears the struct on the radio, but Reset also sends an "unbind" command to the receiver. Obviously if we do it in companion only Delete is possible

**Comment by raphaelcoeffic** [comment]
> Delete just clears the struct on the radio

Ok, then let's have it!

**Comment by Allerhopp** [comment]
Here is my EEPROM backup (carbo from chat) with the broken telemetry (companion).
Names, sensor id ... also sensor instance in radio x9d is wrong with 3 digits instead of 2. Or is it a new feature, i do not understand .....
[eeprom-2019-08-18-190908.zip](https://github.com/opentx/opentx/files/3513432/eeprom-2019-08-18-190908.zip)

N064 X9D I did a new sensor search in the default model R9M to be sure.

**Comment by raphaelcoeffic** [comment]
@Allerhopp there is really a lot of problems with Companion and X9D+ at the moment! Thx for pointing at it! It will be fixed shortly. I've found already a couple of issues, but there are more...

File: companion/src/firmwares/opentx/opentxeeprom.h [hunk 1]
```diff
@@ -83,7 +83,8 @@ class ProtocolsConversionTable: public ConversionTable
       addConversion(PULSES_PXX_XJT_D8, val);
       addConversion(PULSES_PXX_XJT_LR12, val++);
 
-      addConversion(PULSES_ACCESS_ISRM, val++);
+      addConversion(PULSES_ACCESS_ISRM, val);
```
Comment (bsongis) [inline companion/src/firmwares/opentx/opentxeeprom.h:86]: I wonder if my previous code didn't already break the module type conversion (i.e. a DSM2 module)

**Pull request merged**